internal/app/productinfo/api/handlers.go

```diff
@@ -0,0 +1,104 @@
+package api
+
+import (
+	"net/http"
+
+	"fmt"
+	"github.com/gin-gonic/gin"
+	log "github.com/sirupsen/logrus"
+)
+
+// getServices retrieves services supported by the gibven provider in the given region
+func (r *RouteHandler) getServices(c *gin.Context) {
+
+	pathParams := getPathParams(c)
+	log.Debug("Path params: [%s]", pathParams)
+
+	var err error
+	if infoer, err := r.prod.GetInfoer(pathParams.Provider); err == nil {
+		if services, err := infoer.GetServices(pathParams.Region); err == nil {
+			c.JSON(http.StatusOK, services)
+			return
+		}
+	}
+
+	c.JSON(http.StatusInternalServerError, fmt.Sprintf("error while retrieving services: %s", err.Error()))
+
+}
+
+func (r *RouteHandler) getService(c *gin.Context) {
+
+	pathParams := getPathParams(c)
+	log.Debug("Path params: [%s]", pathParams)
+
+	c.JSON(http.StatusInternalServerError, "not yet implemented")
```

Comment:
Just a semantical thing, but there is an http status code for this: 501 Not Implemented